core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -3196,8 +3217,8 @@ private[spark] object Utils extends Logging {
         entry = in.getNextEntry()
       }
       in.close() // so that any error in closing does not get ignored
-      logInfo(s"Unzipped from $dfsZipFile\n\t${files.mkString("\n\t")}")
     } finally {
+      logInfo(s"Unzipped from $dfsZipFile\n\t${files.mkString("\n\t")}")
```

Comment:
This log should be printed after successful decompression. It doesn't seem appropriate to put it in finally